docker/action.yml

```diff
@@ -0,0 +1,55 @@
+name: "quarto-docker"
+description: "Build and push a Quarto Docker container to the GitHub Container Registry."
+inputs:
+  source:
+    description: "Full path to quarto file"
+    required: true
+  version:
+    description: "The version to tag the image with."
+    required: true
+  token:
+    description: "The token to authenticate to docker registry"
+    required: true
+  username:
+    description: "The username to authenticate to docker registry"
+    required: true
+  org:
+    description: 'The GitHub orginization that will host Docker image'
+    required: true
+  name:
+    description: 'The name of the Docker image'
+    required: true
+  daily:
+    description: "Apply the `daily` tag to the image."
+    required: false
+    default: true
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: echo "BASE_NM=ghcr.io/${{ inputs.org }}/${{ inputs.name }}" >> $GITHUB_ENV
+
+    - name: Create DockerFile
+      shell: bash
+      run: |
+        cat << 'EOF' >> Dockerfile
+        FROM ubuntu:22.04
+        COPY ${{ inputs.source }} quarto-linux-amd64.deb
+        RUN dpkg -i quarto-linux-amd64.deb && rm quarto-linux-amd64.deb
+        EOF
+
+    - if: ${{ inputs.daily != 'true' && inputs.daily != 'True'  }}
```

Comment:
[`fromJSON()`](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson) should be able to convert an input string `true` to boolean and allow `fromJSON(input.daily) == false` - it would be simpler. 

In composite action, inputs are strings always, but if we fix the `workflow_dispatch` with typed input, this should be ok. 

Non blocking but simpler to understand and maintain probably.

Reply:
Yeah I saw some GitHub issues with composite actions not accepting booleans.  That's why I stayed away from this

see [this issue](https://github.com/actions/runner/issues/2238)

Reply:
Oh I see what you mean if it is boolean it will always be lowercase.   Sorry just realized what you were saying.

Reply:
I'll hold off on changing this until we get a comment on changing the inputs to boolean

Reply:
> Oh I see what you mean if it is boolean it will always be lowercase.

yes exactly - still be a string but either 

* `fromJSON(input.daily)` would get you a boolean
* `inputs.daily` could be compared to string `'true'` or `'false'` 

Less case to test. We can do the change with the workflow dispatch

Reply:
OK I made the change in https://github.com/quarto-dev/quarto-cli/pull/5118/commits/d82b4d660d970ad22cd88dc98806aba19013bba8